c2corg_ui/templates/utils/__init__.py

```diff
@@ -82,3 +82,30 @@ def has_associations(doc):
         a.get('all_routes') or a.get('routes') or \
         a.get('recent_outings') or a.get('outings') or \
         a.get('articles') or a.get('xreports') or a.get('books')
+
+
+def get_route_gear_articles(route):
+    articles = []
+    if route.get('activities'):
+        if ('snowshoeing' in route.get('activities') or
```

Comment:
Since ``route.get('activities')`` is used in all conditions, I recommend to set a variable instead ``activities = route.get('activities')``.

Then you can write:
```
if 'snowshoeing' in activities or 'skitouring' in activities:
```

A very pythonic way to write this condition would be to use list comprehension:
```
if any(activity in activities for activity in ['snowshoeing', 'skitouring']):
```
but perhaps it doesn't really bring a readability gain here?
For ``any()`` see https://docs.python.org/3/library/functions.html#any (the "contrary function" is ``all()``).